pandas/tests/test_tseries.py

```diff
@@ -737,6 +737,13 @@ def test_get_period_field_raises_on_out_of_range(self):
     def test_get_period_field_array_raises_on_out_of_range(self):
         self.assertRaises(ValueError, period.get_period_field_arr, -1, np.empty(1), 0)
 
+class TestDaysInMonth(tm.TestCase):
+    def test_day_not_in_month_coerce_true(self):
+        self.assertTrue(isnull(to_datetime('2015-02-29', coerce=True)))
+        self.assertTrue(isnull(to_datetime('2015-02-29', format="%Y-%m-%d", coerce=True)))
+        self.assertTrue(isnull(to_datetime('2015-02-32', format="%Y-%m-%d", coerce=True)))
+        self.assertTrue(isnull(to_datetime('2015-04-31', format="%Y-%m-%d", coerce=True)))
+
```

Comment:
can you add off these cases again, but with ``coerce=False`` (and assert that they raise ``ValueError``)